pages/blog/release-notes-2.5.0.md

```diff
@@ -0,0 +1,147 @@
+---
+title: AsyncAPI Spec 2.5.0 Release Notes
+date: 2022-09-29T16:41:00+01:00
+type: Communication
+tags:
+  - Specification
+  - Release Notes
+cover: /img/posts/release-notes-2.5.0/cover.webp
+authors:
+  - name: Vladimír Gorej
+    photo: /img/avatars/char0n.webp
+    link: https://twitter.com/vladimirgorej
+    byline: Open Source Software Engineer, OSS contributor, author and content creator.
+excerpt: "AsyncAPI 2.5 is now released. This brings really helpful additions, such as extended Server Variables reusability, and many clarifications"
```

Comment:
```suggestion
excerpt: "The release of AsyncAPI 2.5 brings helpful additions, such as extended Server Variables reusability and many clarifications."
```

Never _start_ or _end_ a sentence with an **antecedent** (.ie. `this`) because it's grammatically undefined what `this` references.